### PR TITLE
(Docs): Updated various fields in the `p5.MonoSynth` reference

### LINKED ISSUE #727: Docs : missing description for various fields of `p5.MonoSynth` reference
In the `p5.MonoSynth`  reference, descriptions of some fields are not provided like: `decay`, `sustain`, and `release`

refer: https://p5js.org/reference/#/p5.MonoSynth

image

### PR DESCRIPTION
**Pull request by Abhijay007**
Signed-off-by: Abhijay Jain <Abhijay007j@gmail.com>

added description for the various fields of `p5.MonoSynth reference` such as `decay`, `sustain` and `release`

resolves #727

### CODE REVIEW

**Comment by Abhijay007** [comment]
cc : @davepagurek

File: src/monosynth.js [hunk 1]
```diff
@@ -77,14 +77,21 @@ class MonoSynth extends AudioVoice {
      * @for p5.MonoSynth
      */
     /**
+     * allow user to set the decay time of the envelope (ADSR) of the MonoSynth class.
```
Comment (davepagurek) [inline src/monosynth.js:80]: Tiny change: this should probably be "Allows" for consistency

File: src/monosynth.js [hunk 2]
```diff
@@ -77,14 +77,21 @@ class MonoSynth extends AudioVoice {
...
+     * It is a getter and setter that can be used to retrieve or change the decay time.
+     * used in conjunction with the attack, sustain, and release fields/functions to set the full envelope of the synthesizer.
```
Comment (davepagurek) [inline src/monosynth.js:82]: For here and elsewhere in these docs, we should probably be consistent and capitalize the starts of sentences

**Review by davepagurek — COMMENTED** [review]
The text looks good! Just some minor capitalization/typo changes then we're good to go!

**Review by davepagurek — APPROVED** [review]
Thanks for the updates!

**Pull request merged**